node/pkg/governor/governor.go

```diff
@@ -132,6 +135,7 @@ func newTransferFromDbTransfer(dbTransfer *db.Transfer) (tx transfer, err error)
 // ensure that the Governor usage cannot be lowered due to malicious or invalid transfers.
 // - the Value must be negative (in order to represent an incoming value)
 // - the TargetChain must match the chain ID of the Chain Entry
+// - the flow cancel value must always be less than the big transfer limit
```

Comment:
What if we put out a release that reduces the big transfer size and there is something in the 24 hour window that exceeds the new BTS? Would that violate this constraint? I tried looking through the code, but I don't see where that's handled. I could've missed something though.

Reply:
I guess I wasn't as concerned about "governor wonkiness" like the limits being a bit messed up until it clears. I just wanted to make sure we wouldn't panic or kill the processor in that case. If you feel that's covered, okay. (Or maybe have a test to verify that. :grin:).

Reply:
Ah got it. The only remaining case where the processor should restart is here: https://github.com/wormhole-foundation/wormhole/pull/4016/files#diff-f68b867da294e995f0449fff8116515f60d0e718da580251c6fab60f5c4ce4abR277

This represents the sum of the transfers overflowing the bounds of int64